apps/chat/channels.py

```diff
@@ -704,6 +705,10 @@ def _unsupported_message_type_response(self) -> str:
             session=self.experiment_session, experiment=self.experiment, trace_service=self.trace_service
         )
         trace_info = TraceInfo(name="unsupported message", metadata={"message_type": self.message.content_type})
+        chat_message = ChatMessage.objects.create(
```

Comment:
Is there a case where `self.message.message_text` will not be en empty string?

Reply:
I also wondered that but looking at the message parsing code it seems like it could be possible that the text is not empty.